vidarr-cli/src/main/java/ca/on/oicr/gsi/vidarr/cli/TestValidatorScript.java

```diff
@@ -19,15 +21,32 @@ public final class TestValidatorScript extends TestValidator {
   private String outputMetrics;
 
   @Override
-  public Validator createValidator() {
+  public Validator createValidator(String outputDirectory, String id, boolean verboseMode) {
     try {
-      final var tempDir = Files.createTempDirectory("vidarr-test-script");
-      final var calculateScript = tempDir.resolve("calculate");
-      final var calculateDir = tempDir.resolve("output");
-      Files.createDirectories(calculateDir);
-      Files.copy(Path.of(metricsCalculate), calculateScript);
+      // Get current epoch timestamp and format it to date
+      long epoch = System.currentTimeMillis();
+      String date = new SimpleDateFormat("yyyy-MM-dd_HH:mm:ss").format(new Date(epoch));
```

Comment:
Should this be moved up into "call" so that each validator writes its calculate.output/ into the same directory?